src/labkey/constants.ts

```diff
@@ -121,13 +121,18 @@ export type LabKey = {
 export interface User {
     avatar: string
     email: string
+    displayName: string
+    id: number
+    phone: string
+    userId?: number
```

Comment:
What's the difference between id and userId?  I think that at least deserves a comment.

Reply:
Added a comment speaking to this.